### PR TITLE
Case 19475: Fix various Oculus Store test failures

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/19475/Be-ready-to-get-in-Oculus-Store-for-PC

Test plan:
- While using Interface in VR mode on a Rift, unplug the HDMI cable.  Interface should quit but not crash.
- While using Interface in VR mode on a Rift, press the Oculus Home button on your controller.  Oculus Home should open.  Press it again to go back to Interface.  Interface should still work.
- The other changes can only be run on Oculus store builds so will be tested later.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14373/HighFidelity-Beta-PR14271-848aefd.apk

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1144,7 +1144,7 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
 
     // set the OCULUS_STORE property so the oculus plugin can know if we ran from the Oculus Store
     static const QString OCULUS_STORE_ARG = "--oculus-store";
-    setProperty(hifi::properties::OCULUS_STORE, arguments().indexOf(OCULUS_STORE_ARG) != -1);
+    setProperty(hifi::properties::OCULUS_STORE, true);// arguments().indexOf(OCULUS_STORE_ARG) != -1);
```
Comment (SamGondelman) [inline interface/src/Application.cpp]: This is temporary for testing.  Remove before merge.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14373/HighFidelity-Beta-PR14271-848aefd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14373/HighFidelity-Beta-Interface-PR14271-848aefd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14373/HighFidelity-Beta-PR14271-848aefd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14373/HighFidelity-Beta-Interface-PR14271-848aefd.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14466/HighFidelity-Beta-PR14271-799db33.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14466/HighFidelity-Beta-PR14271-799db33.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14466/HighFidelity-Beta-Interface-PR14271-799db33.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14466/HighFidelity-Beta-PR14271-799db33.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14466/HighFidelity-Beta-Interface-PR14271-799db33.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14507/HighFidelity-Beta-PR14271-dd96881.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14507/HighFidelity-Beta-PR14271-dd96881.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14507/HighFidelity-Beta-Interface-PR14271-dd96881.exe

**Comment by SamGondelman** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14518/HighFidelity-Beta-PR14271-dd96881.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14518/HighFidelity-Beta-PR14271-dd96881.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14518/HighFidelity-Beta-Interface-PR14271-dd96881.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14518/HighFidelity-Beta-PR14271-dd96881.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14518/HighFidelity-Beta-Interface-PR14271-dd96881.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14833/HighFidelity-Beta-PR14271-e20aeb2.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14838/HighFidelity-Beta-PR14271-887c4f1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14838/HighFidelity-Beta-PR14271-887c4f1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14838/HighFidelity-Beta-Interface-PR14271-887c4f1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14838/HighFidelity-Beta-PR14271-887c4f1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14838/HighFidelity-Beta-Interface-PR14271-887c4f1.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pre-Merge Pass @conklin94122 
Pulling the cord does not crash interface, and Oculus buttons responds as expected.

**Pull request merged**